content/docs/2019-06-23-introducing-mlrplayground.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "Introducing mlrPlayground"
+authors: ["sebastian-gruber"]
+date: "2019-06-23"
```

Comment:
FYI: If you date the date back in time, you get a preview rendered by Netlify in the Github PR. 
This is nice for getting an impression on how your post will look like. 
Especially for reviewers who did not do a local rendering.